src/transform/operation.ts

```diff
@@ -8,16 +8,19 @@ export function transformOperationObj(
   operation: OperationObject,
   {
     version,
+    pathItem,
     globalParameters,
   }: {
     version: number;
+    pathItem?: PathItemObject;
     globalParameters?: Record<string, ParameterObject>;
   }
 ): string {
   let output = "";
 
-  if (operation.parameters) {
-    output += `  parameters: {\n    ${transformParametersArray(operation.parameters, {
+  if (operation.parameters || pathItem?.parameters) {
```

Comment:
This line seemed to throw ts-jest. If you could rebase off latest `main` and push again, we should see the tests run.

Reply:
`optional chaining` is not supported by the current typescript settings.

Reply:
Thanks @dnalborczyk. But weird, I obviously ran this locally and `ts-jest` should have used `tsconfig.json` with the correct transpile options. Could this possibly be related to Node 12 being used in CI while I use Node 14?
But no problem changing it obviously.

Reply:
good question. not quite sure. the test run also failed in the ci with node.js v14: https://github.com/drwpow/openapi-typescript/runs/2141031952 . I'd have to pull your PR with that commit and run it locally as well.

Reply:
there seems to be a bug in the ci setup. v14.x and v12.x are essentially running node v10.x

![image](https://user-images.githubusercontent.com/2903325/112235111-5f3d0080-8c14-11eb-861e-d06fcfbd2fc0.png)

![image](https://user-images.githubusercontent.com/2903325/112235130-67953b80-8c14-11eb-97b2-eb95064963df.png)

Reply:
fixed in: https://github.com/drwpow/openapi-typescript/pull/539

Reply:
Thanks so much for improving this.